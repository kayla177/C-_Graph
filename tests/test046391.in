ENTITY 68ZHARD1 name_68ZH journal
ENTITY CFUHM0UY name_CFUH institution
ENTITY 5QJVWFGI name_5QJV author
RELATIONSHIP 68ZHARD1 presented_at CFUHM0UY 9.5
PRINT 68ZHARD1
HIGHEST
EXIT
