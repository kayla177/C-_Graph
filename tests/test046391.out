success
success
success
success
CFUHM0UY
68ZHARD1 CFUHM0UY 9.5
